### PR TITLE
Drop Symfony 2.1 support

### PR DESCRIPTION
**Pull request by mpdude**
Yes, it was a great time, but let's move on.

### CODE REVIEW

**Comment by mpdude** [comment]
Wait, will be pushing more commits.

@benja-M-1 what's the best way to notify you? Are you watching this repo?

**Comment by mpdude** [comment]
*ping* @benja-M-1

File: Resources/config/services.xml [hunk 1]
```diff
@@ -17,12 +17,14 @@
     </parameters>
 
     <services>
-        <service id="bg_object_routing.adapter.symfony_21" class="%bg_object_routing.adapter.symfony_21.class%">
+        <service id="bg_object_routing.adapter.symfony_22" class="JMS\ObjectRouting\Symfony\Symfony22Adapter" public="false">
```
Comment (benja-M-1) [inline Resources/config/services.xml]: @mpdude can you use a parameter instead to stay consistent with the rest of the code? You can replace ```bg_object_routing.adapter.symfony_21.class``` by yours.

Thanks

**Review by benja-M-1 — CHANGES_REQUESTED** [review]

File: Resources/config/services.xml (hunk 1 above)
Comment (mpdude) [inline Resources/config/services.xml]: I removed the parameter as it was my understanding that this approach was a mistake during the early days of Symfony that has meanwhile been discouraged.

But I will change it later if you'd like to keep it.

**Comment by mpdude** [comment]
@benja-M-1 updated

**Comment by benja-M-1** [comment]
Thanks @mpdude

**Comment by mpdude** [comment]
Thank you!

Could you tag a new minor release please?

**Pull request merged**